### DISCUSSION TITLE
Need assistance/guidance for nRF24 based library?

### DISCUSSION THREAD

**Discussion by Dark1886** [zephyrproject-rtos/zephyr#53848] [Q&A]
Is anybody aware of a library/module/etc that supports the nRF24L01+ module? I've seen some nordic nRF libraries, but those don't seem to support this specific module. Currently I only see some Arduino libraries that support it but I expect that porting process would be rather difficult.

**Comment by chrisco484** [comment]
I was wondering the same thing. Those nRF24L01+ modules are really popular and I've been using them in the Arduino world for years but now looking at moving to Zephyr as my projects have become non trivial and so an RTOS is appropriate.
